app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,49 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
+
+  input {
+    -moz-appearance: none;
+    -ms-appearance: none;
+    -webkit-appearance: none;
+    border: 1px solid #E3E3E3;
+    border-radius: 10px;
+    vertical-align: middle;
+    padding: 4px;
+    outline: none;
+    height: 20px;
+    width: 20px;
+    margin: 10px;
+    background: #FFFFFF;
+  }
+
+  html[dir='ltr'] & {
+    text-align: left;
+  }
+
+  html[dir='rtl'] & {
+    text-align: right;
+  }
+
+  label {
+    display: block;
+
+    input:checked {
+      background: #0097D0;
+      background-clip: content-box;
+    }
+  }
+  label:hover {
+    input {
+      border: 1px solid #0097D0;
+    }
+  }
+
+  label:active {
+    input {
+      box-shadow: #55B8EB 0px 0px 0px 2px
```

Comment:
this colour needs to use existing or new variables as I mentioned above. Example: https://github.com/mozilla/fxa-content-server/blob/c3cc9d4a462e9c96cd702a5e0161065c4d8c0139/app/styles/_layout.scss#L25